scripts/dupe_finder.py

```diff
@@ -0,0 +1,63 @@
+import json
+import os
+import hashlib
+
+
+def getFileHash(filename, alg, chunksize=131072):
+    if (alg == 'sha256'):
+        h = hashlib.sha256()
+    elif (alg == 'sha1'):
+        h = hashlib.sha1()
+    elif (alg == 'md5'):
+        h = hashlib.md5()
+
+    with open(filename, 'rb', buffering=0) as f:
+        for b in iter(lambda: f.read(chunksize), b''):
+            h.update(b)
+    return h.hexdigest()
+
+
+'''
+    check if rom is present in the list of files in json
+    return: -1 if no rom is listed, rom's filename if found
+'''
+
+
+def lookForRom(files):
```

Comment:
Can you add an explanation of what this function does?

Reply:
In JSON we have multiple files for each rom. This could lead sometimes to not have any executable file in that list.
For this reason, I've created this function: foreach file in "files" field in JSON, take the extension. If that file is a homebrew, returns its path, otherwise, if files specified in the list of files in the JSON does not contain any executable gb file, return -1.

Reply:
(if you were referring to getFileHash: it's not my code, I simply copy-pasted from hashes. It evaluates hash of a file)

Reply:
I was referring to lookForRom, thanks for the explanation. Can you add it there as a comment?